docs/api-reference/global-functions/Float.md

```diff
@@ -1,17 +1,27 @@
 ---
-description: Float Float(Float value)
+uid: crmscript_global_float
+title: Float
+description: CRMScript Float constructor
 intellisense: Void.Float
 langref: 1
-keywords: Float(Float)
+keywords: Float(Float), Float(String), Float(Integer)
 so.topic: reference
 ---
 
```

Comment:
This overall description does not seem to follow the same conventions established in other crmscript classes docs.

Perhaps link to, or include examples from [Float types documentation](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/float-type.md).